packages/replayio/src/utils/initialization/initialize.ts

```diff
@@ -47,34 +43,11 @@ export async function initialize({
     accessToken = await promptForAuthentication();
```

Comment:
Agreed. This is still the part of this that I'm thinking about the most.

Reply:
One idea that occurs to me is that this task can pause/resume the timeout logic.

That would mean that events would queue behind a successful or failed auth attempt, which seems okay because they'd be flushed before close (even in the failed auth attempt).

The case that would fall between the cracks, I think, would be a permanently hung auth attempt (neither success nor failure) which we could maybe handle by adding an explicit timeout to the `promptForAuthentication` method too... thoughts?

Reply:
Another idea (that I would need to think about more) is that we could get rid of the timeout part and just queue all logs behind auth _except_ if `flushAndClose` is called (pre-exit) in which case we could send everything then.

That would not cover the case where a user force-quits before auth-ing but that's maybe not super common?